### PR TITLE
fix: update pathways commit

### PR DESCRIPTION
**Pull request by pkulkark**
## Description

This PR updates the commit ID for lx-pathway-plugin to the latest one containing the drop table migration.

OpenCraft Internal Jira Ticket: [FAL-3510](https://tasks.opencraft.com/browse/FAL-3510)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pkulkark! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by awais786** [comment]
Right now mysql8 backup going on. May be we can wait because this PR will delete `lx-pathway` tables from db.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -584,7 +584,7 @@ EDXAPP_PRIVATE_REQUIREMENTS:
     - name: git+https://github.com/open-craft/labxchange-xblocks.git@a0a8a8dad13199014d4bb29cee416289880bde0b#egg=labxchange-xblocks
       extra_args: -e
     # "Pathways" learning context plugin for the LabXchange project
-    - name: git+https://github.com/open-craft/lx-pathway-plugin.git@ba1d470217cd5908cbd8b56075628bd4eacf7b39#egg=lx-pathway-plugin
+    - name: git+https://github.com/open-craft/lx-pathway-plugin.git@ec6778e51b0b56056669c43c32f1e6c45f97359f#egg=lx-pathway-plugin
```
Comment (awais786) [inline playbooks/roles/edxapp/defaults/main.yml:587]: This is not a valid path.

Comment (awais786) [reply playbooks/roles/edxapp/defaults/main.yml:587]: @pkulkark  You need to change the complete path from opencraft to labxchange.

https://github.com/labxchange/lx-pathway-plugin/pull/17

**Comment by mphilbrick211** [comment]
Hi @pkulkark! Flagging the comments above, and the branch conflicts that have popped up.

**Comment by mphilbrick211** [comment]
> Hi @pkulkark! Flagging the comments above, and the branch conflicts that have popped up.

Friendly follow-up on this @pkulkark!

**Comment by mphilbrick211** [comment]
Closing this for now - can reopen if needed!

**Comment by openedx-webhooks** [comment]
@pkulkark Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/configuration&entry.867055334=https://github.com/openedx/configuration/pull/7025&entry.1484655318=https://github.com/pkulkark&entry.752974735=2023-09-26+17:46&entry.1917517419=2024-02-21+14:47&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**